gpflow/models/vgp.py

```diff
@@ -66,10 +67,10 @@ def __init__(
             num_latent_gps = self.calc_num_latent_gps_from_data(data, kernel, likelihood)
         super().__init__(kernel, likelihood, mean_function, num_latent_gps)
 
-        X_data, Y_data = data
+        self.data = data_input_to_tensor(data)
+        X_data, Y_data = self.data
```

Comment:
```suggestion
        self.data = X_data, Y_data = data_input_to_tensor(data)
```

Reply:
Double assign is dirty style and unambiguous with comma operator. This approach is prone to mistakes.

Reply:
```suggestion
        self.data = (X_data, Y_data) = data_input_to_tensor(data)
```
?

Reply:
Still, having two separate lines is preferable.